src/main/java/com/nirima/jenkins/plugins/docker/DockerContainerWatchdog.java

```diff
@@ -107,6 +114,15 @@ protected boolean stopAndRemoveContainer(DockerAPI dockerApi, Logger logger, Str
      * Implementation of business logic
      */
     
```

Comment:
General code formatting issue:  Can you remove trailing whitespace from all lines of code you're adding or changing?
e.g. Do a regex search-and-replace to replace "space space asterisk dollar" with the empty string?

It's a personal pet-hate of mine, but it's not _just_ [OCD](https://en.wikipedia.org/wiki/Obsessive%E2%80%93compulsive_disorder) ;-) - it can also cause merge issues when people accidentally add or remove whitespace (e.g. their IDE does this without their knowledge) because it makes git (etc) think that lines have been changed when they haven't.  If we always ensure that lines don't have random trailing whitespace then it's easy to prevent this being an issue.